content/advanced-operations/target-word-collocations.Rmarkdown

```diff
@@ -35,7 +35,7 @@ toks_inside <- tokens_remove(toks_inside, pattern = eu) # remove the keywords
 toks_outside <- tokens_remove(toks_news, pattern = eu, window = 10)
 ```
 
-We compute words' association with the keywords using `textstat_keyness()`.
+We will compute words' association with the keywords using `textstat_keyness()`.
```

Comment:
But this is probably "can" because this is more general statement about `textstat_keyness()`.